rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -30,7 +30,7 @@ function exit_handler()
 {
   printf "%s %s\n" \
     "[DEBUG]" "Yarn container exit handler: EXIT signal received in yarn container, exiting ..."
-  pid=$(docker inspect --format={{{ inspectFormat }}} $docker_name) && kill -9 $pid ||\
+  pid=$(docker inspect --format={{{ inspectFormat }}} $docker_name 2>/dev/null) && kill -9 $pid ||\
     printf "%s %s\n"  "[DEBUG]" "Yarn container exit handler tries to kill the container $docker_name that does not exist. It probably has already exited."
```

Comment:
>printf "%s %s\n [](start = 4, length = 15)

I suggest to break this into multiple steps.
1. get pid (and drop the log in case of failure)
2. if cannot get pid, go to 4
3. if get the pid, kill -9. log the result.  
4. kill 0